astroquery/mpc/core.py

```diff
@@ -0,0 +1,296 @@
+# -*- coding: utf-8 -*-
+
+from ..query import BaseQuery
+from . import conf
+from ..utils import async_to_sync
+
+__all__ = ['MPCClass']
+
+
+@async_to_sync
+class MPCClass(BaseQuery):
+    MPC_URL = 'http://' + conf.server + '/web_service/search_orbits'
+    # The authentication credentials for the MPC web service are publicly available and
+    # can be openly viewed on the documentation page at https://minorplanetcenter.net/web_service/
+    MPC_USERNAME = 'mpc_ws'
+    MPC_PASSWORD = 'mpc!!ws'
+
+    def __init__(self):
+        super(MPCClass, self).__init__()
+
+    def query_object_async(self, *args, **kwargs):
+        """
+        Query around a specific object within a given mission catalog
+
+        The following are valid query parameters for the MPC API search. The params list and
+        description are from https://minorplanetcenter.net/web_service/ and are accurate
+        as of 3/6/2018
+
+        Parameters
+        ----------
+        **kwargs :
```

Comment:
please drop `**kwargs`` here, and just list everything as optional parameters (as it's done in the other modules). Also, please list all `*args` the method takes.